### PR TITLE
Switch to PyJWT and get rid of conflicts with jwt library.

### PR DESCRIPTION
**Pull request by bart84ek**

### CODE REVIEW

**Comment by szewczykmira** [comment]
@bart84ek CI is failing

File: land-queue/readme.md [hunk 1]
```diff
@@ -42,7 +42,8 @@ App can be to one repo or all repositories where user is admin.
 ## Requirements
 
 - [Oya](https://oya.sh/)
-- Python 3.8.2 + pyenv
+- Python 3.8.2
+- pip-tools (`$ pip install pip-tools`)
```
Comment (szewczykmira) [inline land-queue/readme.md:46]: Pip tools are installed via `oya run install` so I'm not sure they must be here

Comment (bart84ek) [reply land-queue/readme.md:46]: It was missing on my computer.
What installs them, i can't find it in `requiremts` files

Comment (szewczykmira) [reply land-queue/readme.md:46]: Well, that's a bug then :)

**Pull request merged**